node/src/Pos/WorkMode/Class.hs

```diff
@@ -94,13 +96,11 @@ type WorkMode ssc ctx m
       , HasLens' ctx StartTime
       , HasLens' ctx StateLock
       , HasLens' ctx StateLockMetrics
-      , HasLens' ctx LrcContext
-      , HasLens' ctx UpdateContext
-      , HasLens' ctx UpdateParams
-      , HasLens' ctx SecurityParams
-      , HasLens' ctx TxpGlobalSettings
-      , HasLens' ctx GenesisUtxo
-      , HasLens' ctx GenesisWStakeholders
+      , HasLens LrcContext ctx LrcContext
```

Comment:
Mda. There was a dedicated PR that did `HasLens => HasLens'` and you merged it back like this. 
nvm i guess

Reply:
:trollface: